sdks/java/io/elasticsearch/src/main/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIO.java

```diff
@@ -203,7 +202,9 @@
 })
 public class ElasticsearchIO {
```

Comment:
please remove all leftover references to shardPreference, it was an ES2 only code